assets/js/components/notifications/SetupSuccessBannerNotification.stories.js

```diff
@@ -89,19 +89,6 @@ PageSpeedInsights.scenario = {
 	label: 'Global/SetupSuccessBannerNotification/PageSpeedInsights',
 };
 
-export const IdeaHub = Template.bind( {} );
-IdeaHub.storyName = 'Authentication Success - Idea Hub';
-IdeaHub.parameters = {
-	module: { slug: 'idea-hub', name: 'Idea Hub' },
-	query: {
-		notification: 'authentication_success',
-		slug: 'idea-hub',
-	},
-};
-IdeaHub.scenario = {
```

Comment:
@sashadoes Since we are removing this story, can you make sure the VRT reference images are removed as well? For eg:

- https://github.com/google/site-kit-wp/blob/enhancement/6235-remove-idea-hub/tests/backstop/reference/google-site-kit_Global_SetupSuccessBannerNotification_IdeaHub_0_document_0_small.png
- https://github.com/google/site-kit-wp/blob/enhancement/6235-remove-idea-hub/tests/backstop/reference/google-site-kit_Global_SetupSuccessBannerNotification_IdeaHub_0_document_1_medium.png
- https://github.com/google/site-kit-wp/blob/enhancement/6235-remove-idea-hub/tests/backstop/reference/google-site-kit_Global_SetupSuccessBannerNotification_IdeaHub_0_document_2_large.png